docs/first-steps/post-deployment.md

```diff
@@ -138,6 +143,33 @@ Similarly to the management user's MFA enabling step, you are switching from usi
 
 As a last step you need to make sure that `MFA_ENABLED` is set to `true` in the `build.env` file.
 
+## Re-Configure profiles  MFA
+
+For everything to work as expected, you need to set the aws configuration profiles to the corresponding mfa-device configuration.
+
+Then run `leverage credentials configure` again with the `--fetch-mfa-device` flag and select the `skip credentials configuration` option.
+
+Also, the flag `--skip-access-keys-setup` could be used instead to avoid the interactive step!(https://leverage.binbash.com.ar/user-guide/leverage-cli/reference/credentials/#options)
```

Comment:
```suggestion
As an alternative, the flag [`--skip-access-keys-setup`](https://leverage.binbash.com.ar/user-guide/leverage-cli/reference/credentials/#options) could be used to avoid the interactive step.
```

Reply:
done!